packages/app/src/elements/app.ts

```diff
@@ -547,7 +547,7 @@ export class App extends ServiceWorker(StateMixin(AutoSync(ErrorHandling(AutoLoc
         target.classList.add("dragging");
         target.addEventListener("dragend", () => target.classList.remove("dragging"), { once: true });
         const totp: TOTPElement | null = (target.querySelector("pl-totp") ||
-            (target.shadowRoot && target.shadowRoot.querySelector("pl-totp"))) as TOTPElement | null;
+            target.shadowRoot?.querySelector("pl-totp")) as TOTPElement | null;
```

Comment:
Refactored just because I saw it.

Reply:
Good catch! I learned about the optional chaining syntax not so long ago so there are still quite a few places with the "old" pattern.